css/portal-dashboard/custom-select.less

```diff
@@ -7,8 +7,6 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    width: 200px;
-    height: 30px;
```

Comment:
From what I can tell we still need this `height: 30px;` to match the spec.  Otherwise the header of the select shrinks vertically. Studying the spec, I think we also need a `box-sizing: border-box;` to get it to be pixel perfect.  So I think this class needs:
```
height: 30px;
box-sizing: border-box;
```